lib/phoenix/verified_routes.ex

```diff
@@ -256,7 +304,9 @@ defmodule Phoenix.VerifiedRoutes do
   @doc false
   def __verify__(routes) when is_list(routes) do
     Enum.each(routes, fn %__MODULE__{} = route ->
-      unless match_route?(route.router, route.test_path) do
+      test_path = split_test_path(route.test_path)
+
+      unless route.router.verified_route?(route.router, test_path) do
```

Comment:
Empty plug_options, i.e. a list, would probably be a "safer" default?

```suggestion
      unless route.router.verified_route?([], test_path) do
```

Reply:
Ah, actually I got that wrong. It should be `route.plug_opts`, as we do have that information here.

Reply:
nope, nevermind, we don't. 🤔 That does seem like pretty important information for plugs to verify their routes?

Reply:
Okay, this is a different thing, nvm 😄.